README.md

```diff
@@ -0,0 +1,4 @@
+# deliveryFee
+
+
```

Comment:
In the actual submit scenario, you can mention execution instructions here. 
Together with a DockerFile for building and running your service locally.
Other than that, a bash script containing a curl that can verify if your service is working would also be useful.